PocketBudgetTracker/install_deps.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
```

Comment:
Maybe it's better to put this file into `tools` dir?

Reply:
And create `setup.py` here?

Reply:
And don't use `requirements.txt` here (as all the dependencies can be put into the `setup.py`)?

Reply:
I thought about moving it into tools folder. But current implementation of install_deps uses current folder ( where this script located) to search for submodules otherwise the code looks inconvenient
_modules_dir = os.path.join(os.path.dirname(__file__), "../PocketBugetTracker")_

Reply:
Also I do not want to care about of searching for errors if we decide to rename the folder *Pocket BugTracker*, for example